onadata/apps/logger/models/instance.py

```diff
@@ -391,7 +391,15 @@ def update_project_date_modified(instance_id, _):
         if current_task.request.id:
             raise e
     else:
-        instance.xform.project.save(update_fields=["date_modified"])
+        # update project date_modified using raw SQL
+        with connection.cursor() as c:
+            sql = (
+                "UPDATE logger_project SET "
+                "date_modified = %s "
+                "WHERE id = %s"
+            )
+            params = [timezone.now(), instance.xform.project.pk]
+            c.execute(sql, params)
```

Comment:
Could we use the Queryset approach which also does not trigger an `post/pre_save` signals?
```python
Project.objects.filter(pk=instance.xform.project_id).update(date_modified=timezone.now())
```